### ISSUE TITLE
[Request] Offer Linux based versions

### ISSUE THREAD

**Issue by audio2000** [davy7125/polyphone#209]
Linux / Debian Based

First, thank you so much for this application, really great.
I do not know if this is really possible / something you would want to do based on the nature of Linux packaging.

Linux users do not really get the best access to Polyphone.
-Currently my repo only offers the older version 2.2.0
-The only other alternative is using a Flatpak which I am not be interested in personally and on top of that it is created by a 3rd party user / unverified version.

I would really be interested in being able to help test the new beta version but there is no access to this version for Linux users either.

Probably not, but I was wondering if you would ever consider offering a Debian based .deb version or an AppImage for both the...
Official released version (currently 2.3.0) 
and 
New Beta version being worked on

I have no idea how to do it but I have seen some Linux applications somehow setup automatically generated AppImage versions of their software.  Not sure anything like this would be possible with Polyphone?

Again, I would even want to help test out 2.3.0 and also have some ideas to share but would not want to report issues / suggest features without using the latest versions to eliminate confusion for you.

I would be interested to hear if you have any thoughts on the matter.
Thank you again for creating this application.

**Comment by audio2000** [comment]
@davy7125 
Thank you very much for responding to this.
Ahh the other day there was only a Windows version in the beta section, I see there is a Ubuntu one now.

APPIMAGE
I don't know, I have never seen different versions of AppImages based on distributions, offered by programs that provide AppImages.  The only thing I have seen is programs that have GTK & Qt based versions of the software, then two different AppImage versions are provided.
If only one AppImage had to be created, that would great, Polyphone seems like the perfect application for this format, but you are probably not interested in this regardless.

COMPILING
Eventually I think I am going to try the compiling route because it is the only solution left.
Thank you for providing link how to compile, I guess I will try the Ubuntu based compiling instructions because it is the closest to Debian.

WEBSITE SUGGESTION:
(Not sure if you care / are interested in the following)
I see you have mentioned that you are responsible for the website design so I will mention this.
I actually came across the compiling instructions on your website recently.  I went back to your website before making this post to view them but I could not seem to find them.
With the link you mentioned I now understand why.  It is because if a user clicks "Doc." the website opens up the "Manual" drop down expanded automatically, so user might not see the other two expandable items below (Tutorials & Technical Documentation).  Which is what happened to me, I thought this page was just the manual so I backed out.
Anyways, just trying to be helpful, not sure if this is or not.

Thank you again for your response and for Polyphone in general.

**Comment by davy7125** [comment]
I understand the need for AppImage / FlatPak / Snap packages but this is too much work for me to provide everything.
So what I do is to wait when packages are made by other packagers and then I share the link. I checked now and it appears that a FlatPak package is ready :-)
Here is the link:
https://flathub.org/apps/com.polyphone_soundfonts.polyphone

Yes in the documentation I used a menu with dropdown elements for limiting it's height.

I was fast on some of your suggestions but I'll be longer now since I have other projects in the "real life" to finish now. But I'll be back later.
